src/qt/guiutil.cpp

```diff
@@ -732,6 +734,18 @@ QString formatDurationStr(std::chrono::seconds dur)
     return strList.join(" ");
 }
 
+QString FormatPeerAge(std::chrono::seconds time_connected)
+{
+    const auto time_now{GetTime<std::chrono::seconds>()};
+    const auto age{time_now - time_connected};
+    if (age < 60s) return QObject::tr("%1 s").arg(age.count());
+    const auto hours{std::chrono::duration_cast<std::chrono::hours>(age).count()};
+    if (hours >= 24) return QObject::tr("%1 d").arg(hours / 24);
+    if (hours) return QObject::tr("%1 h").arg(hours);
+    const auto minutes{std::chrono::duration_cast<std::chrono::minutes>(age).count()};
+    return QObject::tr("%1 m").arg(minutes);
```

Comment:
@RandyMcMillan, looking at this further, I think the function in 349c597475d68e90596d165ee1cd5f08bbe59e2b can be simplified and improved as follows:

```diff
@@ -736,12 +736,10 @@ QString FormatPeerAge(std::chrono::seconds time_connected)
 {
     const auto time_now{GetTime<std::chrono::seconds>()};
     const auto age{time_now - time_connected};
-    if (age < 60s) return QObject::tr("%1 s").arg(age.count());
-    const auto hours{std::chrono::duration_cast<std::chrono::hours>(age).count()};
-    if (hours >= 24) return QObject::tr("%1 d").arg(hours / 24);
-    if (hours) return QObject::tr("%1 h").arg(hours);
-    const auto minutes{std::chrono::duration_cast<std::chrono::minutes>(age).count()};
-    return QObject::tr("%1 m").arg(minutes);
+    if (age >= 24h) return QObject::tr("%1 d").arg(age / 24h);
+    if (age >= 1h) return QObject::tr("%1 h").arg(age / 1h);
+    if (age >= 1min) return QObject::tr("%1 m").arg(age / 1min);
+    return QObject::tr("%1 s").arg(age / 1s);
 } 
```

Reply:
I repushed an update to https://github.com/jonatack/gui/commits/gui-peers-tab-age-column with the changes in https://github.com/bitcoin-core/gui/pull/543#discussion_r806000745 and https://github.com/bitcoin-core/gui/pull/543#discussion_r805658460 if you want to cherry-pick them.

Reply:
Should probably add translator notes too

Reply:
> Should probably add translator notes too

Agree, am doing this for several of these (FormatPeerAge, formatDurationStr, TimeDurationField, etc.) as a follow-up.